js/components/Breakpoints.js

```diff
@@ -22,24 +46,12 @@ const Breakpoints = React.createClass({
       gThreadClient.resume();
     }
 
-    function getFilenameFromSources(sources, actor) {
-      const source = sources[actor];
-      const url = new URL(source.url);
-      const filename = url.pathname.substring(
-        url.pathname.lastIndexOf("/") + 1);
-      return filename;
-    }
-
     return dom.div(
       { className: "breakpoints" },
       dom.button({ onClick: onResumeClick }, "resume"),
       dom.ul(
         null,
-        this.props.breakpoints.map(bp => dom.li(
-          null,
-          getFilenameFromSources(this.props.sources, bp.location.actor)
-            + ", line " + bp.location.line)
-        )
+        this.props.breakpoints.map(bp => Breakpoint(this.props.sources, bp))
```

Comment:
You're not actually using a component. You're just using a function. You create a component with `React.createElement(Breakpoint)`. I would just rename `Breakpoint` as `renderBreakpoint` to make it clear that it's just a function.